TaskClass.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Enum representing possible task states
+
+public enum TaskStatus
+{
+	NotStarted,
+	InProgress,
+	Completed,
+	OnHold       // If some issue prevents progression on a task
```

Comment:
Maybe Blocked would be a more meaningful name for this

Reply:
Changed to Blocked